teeio-validator/include/ide_test.h

```diff
@@ -506,7 +526,11 @@ struct _ide_run_test_case {
   ide_run_test_case_t *next;
   char class[MAX_NAME_LENGTH];
   char name[MAX_NAME_LENGTH];
-  // IDE_COMMON_TEST_ACTION action;
+
+  int class_id;
+  int case_id;
+  int assertion_cnt;
```

Comment:
why need assertion_cnt?

Reply:
Yes, assertion_cnt is not needed. It will be deleted in next version.